web/controller/components/forms/BirthDate.js

```diff
@@ -0,0 +1,147 @@
+import React from "react";
+
+import classNames from "classnames";
+import { CURRENT_YEAR } from "common/utils/time";
+import { Input } from "../../../common/forms/Input";
+
+const BIRTH_DATE_MIN_YEAR = 100;
+const BIRTH_DATE_MAX_YEAR = 18;
+
+export function BirthDate({ label, userBirthDate, setUserBirthDate }) {
+  const [day, setDay] = React.useState();
```

Comment:
:green_circle: 
On a ce message d'erreur 
```
Warning: A component is changing an uncontrolled input to be controlled. This is likely caused by the value changing from undefined to a defined value, which should not happen.
```
je pense que c'est dû à l'init de ces states. je sais pas si c'est très grave et si on peut/veut changer ça ?

Reply:
Oui il faudrait que je le change !

- [ ] Corriger "A component is changing an uncontrolled input to be controlled"